src/tests/admin/conferenceroomoutoforderplanning/OutOfOrderIsNotCreatedIfTitleIsEmpty.java

```diff
@@ -0,0 +1,57 @@
+package tests.admin.conferenceroomoutoforderplanning;
+
+import static framework.common.AppConfigConstants.EXCEL_INPUT_DATA;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.util.List;
+import java.util.Map;
+
+import org.json.JSONException;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Test;
+
+import framework.pages.admin.HomeAdminPage;
+import framework.pages.admin.conferencerooms.RoomOutOfOrderPlanningPage;
+import framework.pages.admin.conferencerooms.RoomsPage;
+import framework.rest.RootRestMethods;
+import framework.utils.readers.ExcelReader;
+
+/**
+ * TC18: Verify an Out Of Order cannot be created if title is empty
+ * TC29: Verify that when title is empty the message "Out of order should have a title" is displayed
+ * @author Yesica Acha
+ *
+ */
+public class OutOfOrderIsNotCreatedIfTitleIsEmpty {
+	RoomOutOfOrderPlanningPage outOfOrderPage;
+	ExcelReader excelReader = new ExcelReader(EXCEL_INPUT_DATA);
```

Comment:
Same situation here, All of your variables are used as local variables, but declared as global. Please declare them inside test method